### ISSUE TITLE
RegionAttachment script

### ISSUE THREAD

**Issue by Lgood20** [byrobingames/spinehaxe#1]
Hey Robin,

I've recently updated Stencyl to version 4 (private build 10404) and the Spine2D extension seems to be throwing errors at me.

The first error I got was for the 'RegionAttachment' script, but I have managed to correct with the following line:

"class RegionAttachment extends Attachment #if (haxe_ver < "4.0.0") implements Dynamic<Dynamic> #end  {"

Now it seems to be throwing new errors which has something to do with the 'wrapping' method inside the RegionAttachment script (Image attached):
![2019-07-01_12h32_41](https://user-images.githubusercontent.com/52391978/60407577-ad1f0300-9bfe-11e9-8e27-a8d760d411b9.png)

I was hoping that you may help me with this?

Cheers,

Lindsay

**Comment by byrobingames** [comment]
Hey,

This is a library issue...
This extension is using the library from https://github.com/bendmorris/spinehaxe

What you can try to do is copy spinehaxe folder from  https://github.com/bendmorris/spinehaxe to the spinehaxe extension folder.( STENCYLWORKS/engine-extensions/spinehaxe ).

Run clean project before you compile..

**Comment by Lgood20** [comment]
Hey Robin,

I've try doing that, but it is still coming up with the same issue:
![image](https://user-images.githubusercontent.com/52391978/60550895-d067c080-9d6c-11e9-9b17-ccbcdd6b60fb.png)

Not sure what else to do.

**Comment by Lgood20** [comment]
So I've replaced the entire spinehaxe library with the one here - https://github.com/bendmorris/spinehaxe and now it seems to throw another error (almost as if it doesn't recognize the Spine Blocks you have created).
![image](https://user-images.githubusercontent.com/52391978/60564237-40446e00-9da2-11e9-9297-eb8749734374.png)


EDITED: 
So it looks like it's the blocks in the design mode causing the issue. I had created a blank test game, and enabled the extension (works fine), then, as soon as I add in the blocks from the extension, it causes those issues?
Any thought?
